demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -160,9 +160,23 @@ def to_dict(self) -> Dict[str, Any]:
         Returns:
             Dict[str, Any]: JSON dictionary representation of the class.
         """
-        self.__add_lazy_properties()
+        import inspect
```

Comment:
why importing within the func and not at the top?

Reply:
we can move it to the top